jessie/nginx.conf

```diff
@@ -0,0 +1,32 @@
+user www-data;
+worker_processes  1;
+
+error_log  /var/log/nginx/error.log warn;
+pid        /var/run/nginx.pid;
+daemon off;
+
+events {
+    worker_connections  1024;
+}
+
+
+http {
```

Comment:
Need to add all the right standard configuration params. 
See: https://github.com/maestrano/docker-web-ruby/blob/master/2.3/nginx.conf